man/it.man

```diff
@@ -0,0 +1,61 @@
+.TH VRMS-RPM 1
+.SH NAME
+vrms-rpm - Segnala i software non liberi installati 
+.SH SOMMARIO
+\fBvrms-rpm\fR \fI[OPTION]\fR ...
+.SH DESCRIZIONE
+Questo programma e' un clone del programma per Debian GNU/Linux \fBvrms\fR
+("\fBvirtual Richard M. Stallman\fR"), creato con l'obiettivo di portare la
+sua brillante idea su distribuzioni GNU/Linux basate su RPM.
+.PP
+Questo programma analizza la lista dei pacchetti RPM correntemente installati
+e stampa i pacchetti non liberi sullo stdout. La decisione di classificare un
+pacchetto come libero o non libero e' basata sulle linee guida per le licenze 
+del proggetto Fedora.
+.PP
+Utenti di Fedora / CentOS che usano repository addizionali, come 
+\fBRPM Fusion\fR  noteranno che tutti i pacchetti che arrivano da repository 
+non ufficiali sono segnalati come non liberi. Questo e' causato dal fatto che
+la distinzione, come indicato in precedenza, e' basata sulla licenza del 
+pacchetto. Ci sono altre ragioni, oltre alla licenza, per cui un pacchetto non
+e' stato accettato nel repository ufficiale di Fedora / CentOS - quali patenti
+software o restrizioni all'esportazione.
+.PP
+Richard Stallman e' stato, per anni, la figura piu' importante e piu' vocale
+nel movimento per il Free Software, ricordandoci incessantemente i principi 
+morali riguardanti il software. Come Richard, noi crediamo che gli utenti 
+debbano avere a disposizione un modo semplice per capire se stanno usando 
+software non libero nei loro sistemi. Questo programma e' un tentativo di 
+soddisfare questa esigenza.
+.SH OPTIONS
```

Comment:
The translation uses `[opzione]`. It's best to be consistent.

Reply:
Sorry, I've missed it. Done.